resources/openapi/api-intro.md

```diff
@@ -1,23 +1,9 @@
----
-title: "Metabase API documentation"
----
-
-# Metabase API documentation
-
-_These reference files were generated from source comments by running:_
-
-```
-clojure -M:ee:doc api-documentation
-```
```

Comment:
To be fair you still run the command to generate the dox so I don't know if we need to delete this part

Reply:
True. Was thinking of spacing, but it doesn't look too bad:

![image](https://github.com/user-attachments/assets/262dd4c7-2fbc-4bce-91f2-9a5a5084ab34)

Added it back.